.github/workflows/cleanup.yml

```diff
@@ -0,0 +1,31 @@
+name: Remove old packages
+on:
+  schedule:
+    - cron: "0 8 * * *"
```

Comment:
Sorry, jumped the gun slightly. Can we have it trigger on push to main, like https://github.com/freedomofpress/securedrop-apt-test/blob/main/.github/workflows/cleanup.yml#L3 ? So then cleanup happens instantly and we don't need to coordinate crons, etc.